### PR TITLE
feat(chore): created useCollections composable

### PR DESCRIPTION
**Pull request by craigharman**
## Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [x] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Description
Added all the endpoints for manipulating collections as per the [Directus API](https://docs.directus.io/reference/system/collections/)
You can now list, get (by name), update and delete a collection.
Adds a new composable useDirectusCollection.

## Checklist:

- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes (if not applicable, please state why)

### CODE REVIEW

File: docs/content/en/usage/useDirectusCollections.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
+---
+title: "useDirectusCollections"
+description: "Learn how to use Nuxt & Directus"
+position: 14
+category: "Usage"
+---
+
+> Check out the Directus [Collections](https://docs.directus.io/reference/collections/) documentation.
+
+### `listCollections`
```
Comment (Intevel) [inline docs/content/en/usage/useDirectusCollections.md:10]: I would prefer if we renamed `listCollections` to `getCollections`.

**Review by Intevel — CHANGES_REQUESTED** [review]
Thanks for this PR 💚 Requested a few changes

File: docs/content/en/setup.md [hunk 2]
```diff
@@ -35,6 +35,16 @@ export default {
 };
 ```
 
+If you have a `runtimeConfig` in your `nuxt.config.js` file then you will need to add the `directus` config object to it:
```
[the remark this answers is not in the document]
Comment (craigharman) [reply docs/content/en/setup.md]: No problems. I've removed this.

File: docs/content/en/usage/useDirectusCollections.md (hunk 1 above)
Comment (craigharman) [reply docs/content/en/usage/useDirectusCollections.md]: Renamed as requested - I was just naming it similar to the Directus name: https://docs.directus.io/reference/system/collections/#list-collections

**Review by Intevel — APPROVED** [review]

**Comment by Intevel** [comment]
Please follow the conventional commit rules in the future: https://www.conventionalcommits.org/

**Comment by Intevel** [comment]
And please fix your formatting, with `yarn lint` or `npm run lint` in the root project.

**Comment by craigharman** [comment]
> Please follow the conventional commit rules in the future: https://www.conventionalcommits.org/

Will do, apologies.

**Comment by craigharman** [comment]
> And please fix your formatting, with `yarn lint` or `npm run lint` in the root project.

I ran `npm run lint` but there were no commitable changes?

**Comment by Intevel** [comment]
Thank you. I will merge for the next release.

**Pull request merged**